service/configurer/secret/secret.go

```diff
@@ -140,11 +119,7 @@ func (c *SecretConfigurer) Values() (string, error) {
 		if !ok {
 			return "", microerror.MaskAnyf(keyMissingError, "key '%v' not found in secret", c.key)
 		}
-		m, err := c.decodeModifier.Modify(b)
-		if err != nil {
-			return "", microerror.MaskAny(err)
-		}
-		valuesData = string(m)
+		valuesData = string(b)
```

Comment:
Apparently the byte slice contained in `data` is already a plain string and not base64 encoded as we are used to in `kubectl`.